shared/interfaces/phase0_block_wrapper.go

```diff
@@ -33,7 +33,7 @@ func (w Phase0SignedBeaconBlock) Block() BeaconBlock {
 // IsNil checks if the underlying beacon block is
 // nil.
 func (w Phase0SignedBeaconBlock) IsNil() bool {
-	return w.b == nil
+	return w.b == nil || w.Block().IsNil()
```

Comment:
```suggestion
	return w.b == nil || w.Block() == nil || w.Block().IsNil()
```

Reply:
`w.Block() == nil` would never be true. If you look at our `Block` method, we always use a wrapper around the block proto object. So its impossible for the interface to be nil.